recipes/conda-bash-completion/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   number: 0
-  skip: True  # [win]
+  skip: True  # [win,osx]
```

Comment:
This should be, I think, `[win or osx]`.

Reply:
Wow that worked. I guess you learn something new everyday.

Reply:
he he :smile: 
I'll make it more standard so as to not confuse future maintainers.